src/app/not-found.js

```diff
@@ -1,5 +1,40 @@
+import Link from "next/link";
+{
+  /* the Link tag allows navigation between pages by specifying the path to the desired page  */
+}
+{
+  /* this path is relative to the app folder and is one of files in this folder */
+}
+{
+  /* Next.js preloads the linked page in the background making the next page load almost instantly */
+}
 const NotFound = () => {
```

Comment:
look at other ACM projects to see how they implemented the 404, they are very similar